driver/src/clawfc/CommentTrailingBackslashRecognizer.g4

```diff
@@ -0,0 +1,23 @@
+/*
+ * This file is released under terms of BSD license
+ * See LICENSE file for more information
+ * @author Mikhail Zhigun
+*/
+/**
+ * ANTLR 4 Grammar file for filtering out trailing backslashes at the end of Fortran comment.
+ */
+grammar CommentTrailingBackslashRecognizer;
```

Comment:
Why do we need a grammar file for a simple stuff like that?

Reply:
Makes maintenance easier. This way maintainer only knowledge of basic java and ANTLR would be required for the task.